<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>859</width>
                <height>626</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="createTransaction">
                <property name="geometry">
                    <rect>
                        <x>250</x>
                        <y>530</y>
                        <width>301</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Создать транзакцию</string>
                </property>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>370</y>
                        <width>831</width>
                        <height>96</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QLabel" name="label_6">
                                    <property name="text">
                                        <string>Выберите тип транзакции</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="text">
                                        <string>Выберите категорию транзакции</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QComboBox" name="types">
                                    <property name="enabled">
                                        <bool>true</bool>
                                    </property>
                                    <property name="tabletTracking">
                                        <bool>false</bool>
                                    </property>
                                    <property name="editable">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="deleteType">
                                    <property name="text">
                                        <string>...</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>../icons/delete.png</normaloff>../icons/delete.png
                                        </iconset>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="categories">
                                    <property name="enabled">
                                        <bool>true</bool>
                                    </property>
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="editable">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="deleteCategory">
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>../icons/delete.png</normaloff>../icons/delete.png
                                        </iconset>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                            <item>
                                <widget class="QComboBox" name="insertPolicyType">
                                    <property name="layoutDirection">
                                        <enum>Qt::LeftToRight</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="insertPolicyCategory"/>
                            </item>
                        </layout>
                    </item>
                </layout>
            </widget>
            <widget class="QLineEdit" name="amount">
                <property name="geometry">
                    <rect>
                        <x>360</x>
                        <y>490</y>
                        <width>248</width>
                        <height>22</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="clearButtonEnabled">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>230</x>
                        <y>490</y>
                        <width>131</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Введите количесто</string>
                </property>
            </widget>
            <widget class="QScrollArea" name="scrollArea">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>80</y>
                        <width>801</width>
                        <height>271</height>
                    </rect>
                </property>
                <property name="widgetResizable">
                    <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>0</y>
                            <width>799</width>
                            <height>269</height>
                        </rect>
                    </property>
                    <widget class="QTableView" name="tableView">
                        <property name="geometry">
                            <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>801</width>
                                <height>271</height>
                            </rect>
                        </property>
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                    </widget>
                </widget>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>0</y>
                        <width>801</width>
                        <height>80</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="fullUserName">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="layoutDirection">
                                <enum>Qt::LeftToRight</enum>
                            </property>
                            <property name="autoFillBackground">
                                <bool>false</bool>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
                                    text-decoration: underline;
                                </string>
                            </property>
                            <property name="text">
                                <string>Имя пользователя</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="changeUser">
                            <property name="text">
                                <string>...</string>
                            </property>
                            <property name="icon">
                                <iconset>
                                    <normaloff>../icons/manage_account.png</normaloff>../icons/manage_account.png
                                </iconset>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>859</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <action name="action">
            <property name="text">
                <string>Изменить данные</string>
            </property>
        </action>
        <action name="action_3">
            <property name="text">
                <string>Точно?</string>
            </property>
        </action>
        <action name="action_5">
            <property name="text">
                <string>Добавить категорию</string>
            </property>
        </action>
        <action name="action_7">
            <property name="text">
                <string>Удалить категорию</string>
            </property>
        </action>
        <action name="action_8">
            <property name="text">
                <string>Добавить тип</string>
            </property>
        </action>
        <action name="action_10">
            <property name="text">
                <string>Удалить тип</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
